docs/resources/generated-error-pages/error-codes-inventory.rst.inc

```diff
@@ -1285,6 +1285,22 @@ PACKAGE_NOT_FOUND
 
 
 
+.. _error_code_TEMPLATES_OR_INTERFACES_NOT_FOUND:
+
+TEMPLATES_OR_INTERFACES_NOT_FOUND
+---------------------------------------------------------------------------------------------------------------------------------------
+    
+    **Explanation**: The queried template or interface ids do not exist.
+
+    **Category**: InvalidGivenCurrentSystemStateResourceMissing
+
+    **Conveyance**: This error is logged with log-level INFO on the server side. This error is exposed on the API with grpc-status NOT_FOUND including a detailed error message
```

Comment:
```suggestion
    **Conveyance**: This error is logged with log-level INFO on the server side. This error is exposed on the API with grpc-status NOT_FOUND including a detailed error message.
```

Reply:
Thanks, @carrielaben-da . Will address your comment in a separate PR since it impacts more error codes, not just this one.